src/histolab/filters/morphological_filters.py

```diff
@@ -253,3 +253,27 @@ def __call__(self, np_mask) -> np.ndarray:
 
     def __repr__(self):
         return self.__class__.__name__ + "()"
+
+
+class WhiteTopHat(object):
+    """Return white top hat of an image.
+
+    The white top hat of an image is defined as the image minus its morphological
+    opening. This operation returns the bright spots of the image that are smaller than
+    the structuring element.
```

Comment:
The white top hat of an image is defined as the image minus its morphological
    opening with respect to a structuring element. This operation returns the bright spots of the image that are smaller than
    the structuring element.